tmt-app/src/Components/MajorPage.jsx

```diff
@@ -29,6 +30,31 @@ function MajorPage() {
       >
         {data.name} (Major)
       </Typography>
+      <Grid container>
+        <Grid item>
+          <Box
+            sx={{
+              backgroundColor: "BobaBeige.main",
+              borderRadius: "36px",
+              padding: "2.25rem",
+              margin: {
+                xs: "0.75rem",
+                sm: "1.25rem",
+                md: "1.75rem",
+              },
+            }}
+          >
```

Comment:
Are there any other components out of MUI that we can use here to avoid needing to manually style so much?

Reply:
would be great to take a look at the MUI system and see what else they have to offer just to know. Better to use those components out of the box vs trying to build from scratch when we can.

Reply:
The only mui component that I notice is Card but we would still need to add some custom styling to match our design. However, I did make a custom card because we would need to re-use this.